drivers/common/.gitignore

```diff
@@ -0,0 +1,2 @@
+/_openserial.c
```

Comment:
are you using _openserial.c  and _openserial.h during the debugging? Could you remove it if they are not necessary? Thanks